### PR TITLE
Feature/config local run

### PR DESCRIPTION
**Pull request by Fabex**
For some reasons, a project could not have is serverless.yaml in the project root folder but in a subfolder.
So I add a new argument `--config=` to the local run command in order to config the path of the serverless.yaml file

### CODE REVIEW

File: src/Console/Command/Local.php [hunk 1]
```diff
@@ -77,13 +81,14 @@ public function __invoke(?string $function, ?string $data, ?string $file, ?strin
         return 0;
     }
 
-    private function handlerFromServerlessYml(string $function): string
+    private function handlerFromServerlessYml(string $function, ?string $config): string
     {
-        if (! file_exists('serverless.yml')) {
+        $file = $config ?? 'serverless.yml';
+        if (! file_exists($file)) {
             throw new Exception("No `serverless.yml` file was found to resolve function $function.\nIf you do not use serverless.yml, pass the handler via the `--handler` option: vendor/bin/bref local --handler=file.php");
```
Comment (mnapoli) [inline src/Console/Command/Local.php:88]:
```suggestion
            throw new Exception("No `serverless.yml` file was found to resolve function $function.\nIf you do not use serverless.yml, pass the handler via the `--handler` option: vendor/bin/bref local --handler=file.php\nIf your serverless.yml file is stored elsewhere, use the `--config` option: vendor/bin/bref local --config=foo/serverless.yml");
```

File: docs/local-development.md [hunk 2]
```diff
@@ -63,6 +63,12 @@ $ vendor/bin/bref local --handler=my-function.php --file=event.json
 Hello Jane
 ```
 
+Or if your `serverless.yml` is not in root folder, use the `--config` parameter instead:
+```bash
+$ vendor/bin/bref local hello --config=path/to/your/serverless.yml --file=event.json
+Hello world
+```
```
Comment (mnapoli) [inline docs/local-development.md:70]: I would suggest we keep the documentation simple and not mention the option: instead we suggest it in the error message. That way users will try, fail and be able to fix the command themselves.

**Review by mnapoli — COMMENTED** [review]
Thanks! Just a suggestion regarding the docs and then we're good to merge!

File: src/Console/Command/Local.php (hunk 1 above)
Comment (Fabex) [reply src/Console/Command/Local.php:88]: yes you right

File: docs/local-development.md (hunk 2 above)
Comment (Fabex) [reply docs/local-development.md:70]: ok, no problem, I remove this from the doc

**Comment by mnapoli** [comment]
Thanks!

**Pull request merged**